heroku/github-triage
# Import teams route

## Why the change?

Added /orgs/:org/teams endpoint 

fixes heroku/github-issues-squad/issues/73

## What’s the plan?

- [ ] Write route test
- [ ] Write acceptance test
- [ ] Use Mirage to Mock a 'team' object
- [ ] Write component test

## What does it look like?


Orgs route with clickable links to teams view:
<img width="1229" alt="Screen Shot 2019-06-20 at 10 11 03 AM" src="https://user-images.githubusercontent.com/17086697/59867708-cb097f80-9343-11e9-94f9-8bc5ed023fa5.png">

Teams route
<img width="1254" alt="Screen Shot 2019-06-20 at 10 11 12 AM" src="https://user-images.githubusercontent.com/17086697/59867714-d0ff6080-9343-11e9-8bb5-e75256c62cea.png">


## How do I verify the change?



1. [Visit the review app](...)
2. Click “Foo”
3. Verify you see “Bar”
